src/twisted/newsfragments/11804.bugfix

```diff
@@ -0,0 +1,4 @@
+twisted.web.template.flattenString now recognizes and correctly parses Microsoft Outlook conditional comments. 
```

Comment:
I think that we should keep the release notes brief and describe the change at a very high level.

The information from here is very useful, but I think that it should be moved into our main "narative" documentation.

Something like here https://docs.twisted.org/en/stable/web/howto/twisted-templates.html#quoting

or add a separate small example to describe how the comments are handled.

In this way, if in the future someone has any question about how Twisted XML comments are escaped, it should be easy to discover, without having to read the whole code.... and this code is in a private part of Twisted.

Reply:
got it, I will make the release notes brief, add docstrings where needed (will make a separate test with a proper docstring), and overall explain the work better, but not sure about the narrative documentation part, is that something the community also contributes to, or who handles that?

Reply:
[fe9c8ad](https://github.com/twisted/twisted/pull/11805/commits/fe9c8ad7af50c0085089a5b9d82c357dc58ad1ec)